src/Umbraco.PublishedCache.NuCache/Umbraco.PublishedCache.NuCache.csproj

```diff
@@ -21,12 +21,15 @@
     </ItemGroup>
 
     <ItemGroup>
-        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
Remind me to show you how to do this via MsBuild Targets once we migrate the solution to use src/tests subfolders.